addons/website/models/website_page.py

```diff
@@ -122,9 +123,11 @@ def save_page_info(self, website_id, data):
             'date_publish': data['date_publish'] or None,
             'is_homepage': data['is_homepage'],
             'visibility': data['visibility'],
-            'visibility_password': data['visibility'] == "password" and data['visibility_password'] or '',
             'visibility_group': data['visibility'] == "restricted_group" and data['visibility_group'],
         }
+        if 'visibility_pwd' in data:
```

Comment:
not sure if this condition is needed. In case we have a different visibility, it's good to write on `visibility_password_display` with `''` to make sure we don't have leftover information

Reply:
because if you don't change the password, you don't want to hash the fake password '********'

So if we don't update it, we don't post it to avoid to save ******* has new password.

Reply:
Ok good point but actually I just tested and it doesn't seem to work, after modifying an attribute, the new password is `'********'`. Cf js `dirty` flag comment.